packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -96,9 +96,9 @@ export interface TreeNodeApi {
 
 	/**
 	 * If the given node has an identifier specified by a field of kind "identifier" then this returns the compressed form of that identifier.
```

Comment:
This function now returns the "most compressed" form of the ID, if there is one, so I think we should reword it a bit. Maybe something like:

Returns the {@link FieldKind.Identifier | identifier} of the given node in the most compressed form possible.
@remarks
If the node's identifier is a valid {@link StableNodeKey}, then this will return a unique process-local integer corresponding to that identifier.
If the node's identifier is any other string, then this will return that string.
If the node has no identifier (that is, it has no field of an {@link FieldKind.Identifier | identifier} field kind), then this returns `undefined`.

Reply:
Updated docstring to suggested change